provisioning/inventories/staging/host_vars/beta/vars

```diff
@@ -0,0 +1,29 @@
+---
+# System configuration
+mangaki_name: 'mangaki_staging'
+mangaki_source_install_type: 'package'
+mangaki_source_package: '{{ mangaki_source_home }}/{{ mangaki_source_package_local|basename }}'
```

Comment:
I'd say `python setup.py bdist` but yeah for creating the package file. I'll check what the right variable is for `mangaki_source_package_local` and get back to you.